### DISCUSSION TITLE
roboshop-ansible

### DISCUSSION THREAD

**Discussion by LavanyaRamesh319** [learndevopsonline/learndevopsonline#1890] [Q&A]
I have created roboshop-ansible in jenkins. I am getting error as "frontend.yml could not be found". Could you please help me to resolve this.

workstation public ip address: 54.163.204.173
Jenkins public ip address: 54.91.141.151

git repo : https://github.com/LavanyaRamesh319/roboshop-ansible.git

**Comment by chaithanya1812** [comment]
hi @LavanyaRamesh319 ,
## Issue : - it's not frontend.yml , it roboshop.yml
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/fabdc390-89c4-43d2-a027-b56a5eba9fd1)

**Comment by LavanyaRamesh319** [comment]
It got resolved. Thank you @chaithanya1812 . 

But now I am getting error as "**UNREACHABLE! => {"changed": false, "msg": "Failed to connect to the host via ssh: ssh: Could not resolve hostname frontend-dev.devopz1.online: Name or service not known", "unreachable": true}"**

I checked...hostname also correct.

**Comment by chaithanya1812** [comment]
Hi @LavanyaRamesh319 , This error I experienced morning it's self but I could find your name-servers of your domain is mis-macting
## join the lab session now? It's need your presence. 

![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/186af8a3-d259-486d-b2d3-01f55143e74c)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/ef99f5b2-31d7-4b0e-af0f-0fdabd481b97)

**Comment by LavanyaRamesh319** [comment]
Thank you @chaithanya1812 . Its working fine now
